canonical-web-and-design/www.ubuntu.com
# Add sha256 checksums for desktop and server downloads

## Done

- added sha256 checksums to /download/desktop and /download/server page
- updated the copy docs

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: 
    - http://0.0.0.0:8001/download/desktop
    - http://0.0.0.0:8001/download/server
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to the [desktop copy doc](https://docs.google.com/document/d/1ScuTtdrm3-iAvizTDxksVTAerV6QkCUN8Sg60uUlJsk/edit#) and the [server copy doc](https://docs.google.com/document/d/1FFQQUtzFGH88hA30qnZm-3pwVqcVl9UYyrNT4qQWrSM/edit#)

## Issue / Card

Fixes #4430

## Screenshots

![image](https://user-images.githubusercontent.com/441217/58251604-eccf1100-7d5b-11e9-8dea-05702da30754.png)
